### ISSUE TITLE
cap local externals:setup expects ~/.bucketwise/Capfile to exist

### ISSUE THREAD

**Issue by zoecity** [jamis/bucketwise#1]
Here is the exception I get when running cap local external:setup

```
MacBook-Pro:bucketwise(master) kamal$ cap local externals:setup
/usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:183:in `find_file_in_load_path': no such file to load -- /Users/kamal/.bucketwise/Capfile (LoadError)
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:171:in `load_from_file'
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:89:in `load'
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:86:in `load'
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:86:in `each'
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:86:in `load'
  from ./config/deploy.rb:5:in `load'
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:172:in `load_from_file'
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/configuration/loading.rb:89:in `load'
   ... 13 levels...
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/lib/capistrano/cli/execute.rb:14:in `execute'
  from /usr/local/lib/ruby/gems/1.8/gems/capistrano-2.5.0/bin/cap:4
  from /usr/local/bin/cap:19:in `load'
  from /usr/local/bin/cap:19
```

**Comment by kamal** [comment]
The solution for me was the create the dir and touch the file.

```
$ mkdir ~/.bucketwise
$ touch ~/.bucketwise/Capfile
```

**Comment by jamis** [comment]
Good catch. Fixed in 65c698ea4e4f72083e0b7b84430f7064f072f3eb.
